Makefile

```diff
@@ -0,0 +1,13 @@
+setup:
+	cp .env.example .env
+	composer install
+	php artisan key:generate
+
+run:
+	php artisan serve
```

Comment:
Shouldn't docker be part of running the application?

Reply:
It can be but it does not need to be, for now only for easy available container to startup to project